pygeoapi/filters.py

```diff
@@ -0,0 +1,454 @@
+"""
+For evaluating CQL filter queries from Abstract Syntax Tree
+"""
+
+import logging
+from typing import Tuple
```

Comment:
@jorgejesus `typing` is now standard python, my opinion is that maybe we can start using it gradually. Or our coding style doesn't provide it explicitly and then I have lost this reference. Having it along with all the code is huge for now, it would need a large refactor

Reply:
Yes the `typing` implementation, not the code by it self

Reply:
Ok will change it

Reply:
Done